sql/moz-fx-data-shared-prod/braze_derived/users_v1/query.sql

```diff
@@ -1,84 +1,104 @@
 WITH ctms_emails AS (
   SELECT
-    e.email_id AS external_id,
-    e.primary_email AS email,
-    e.mailing_country,
+    emails.email_id AS external_id,
+    LOWER(emails.primary_email) AS email,
+    LOWER(emails.mailing_country) AS mailing_country,
     CASE
-      WHEN e.double_opt_in = TRUE
+      WHEN emails.double_opt_in = TRUE
```

Comment:
for me the first check would be `has_opted_out_of_emails` = TRUE because that trumps everything?